.github/workflows/continuous-integration-workflow.yml

```diff
@@ -38,23 +38,23 @@ jobs:
 
       # Run tests
       - name: Build and Test
-        uses: GabrielBB/xvfb-action@v1.0
+        uses: GabrielBB/xvfb-action@fe2609f8182a9ed5aee7d53ff3ed04098a904df2 #v1.0
         env:
             NODE_OPTIONS: --max_old_space_size=16384
         with:
           run: npm run test:coverage --silent
 
       # Upload coverage to codecov.io
       - name: Codecov
-        uses: codecov/codecov-action@v1.0.12
+        uses: codecov/codecov-action@07127fde53bc3ccd346d47ab2f14c390161ad108 #v1.0.12
```

Comment:
@fbricon , Minor nitpick but https://github.com/marketplace/actions/codecov is a verified creator. Is this updating the reference regardless of creator status ?

Reply:
> The ‘Verified creator’ badge on GitHub Marketplace is a useful signal, as it indicates that the action was written by a team whose identity has been verified by GitHub. Note that there is risk to this approach even if you trust the author, because a tag can be moved or deleted if a bad actor gains access to the repository storing the action.

Only `actions` and `docker` are trusted for now. I'll ask around if it's OK to relax the req depending on the publisher status